mixins/TabLike.pixy

```diff
@@ -0,0 +1,14 @@
+component TabLike(label string, icon string, track *arn.SoundTrack, user *arn.User)
+	if user == nil
+		a.tab.action(aria-label=label)
+			Icon(icon)
+			span.tab-text= label
+	else
+		if track.LikedBy(user.ID)
+			a.tab.action(data-api="/api" + track.Link(), data-action="unlike", data-trigger="click", aria-label=label)
```

Comment:
This is a little bit risky but not wrong. The link to the resource might theoretically not be the same link for the API, but this is perfectly okay I think.